src/main/kotlin/no/nav/cv/eures/konverterer/esco/JanzzService.kt

```diff
@@ -100,14 +105,22 @@ class JanzzService(
 
         log.info("Query for concept '$conceptId' yielded result $concept in $spentMillis ms")
```

Comment:
Litt utenfor scope for denna PRen, men kan kanskje bruke springs @Timed for å få timing ut til prometheus istedet og måle mer over tid